### DISCUSSION TITLE
HMR is often disconnected after JIT is turned on

### DISCUSSION THREAD

**Discussion by huheng233** [tailwindlabs/tailwindcss#5278] [Help]
I install Tailwind CSS as a PostCSS plugin and turned on JIT mode.

【Precondition】
webpack version: 4.19.1
postcss version: 7.0.36
node version: 12.18.4
tailwindcss version: npm:@tailwindcss/postcss7-compat@^2.2.4

【Problem description】:
When I modify and save the style, the memory used by node will increase by about 0.5G each time, until it reaches the upper limit set by --max-old-space-size, the hot update will be disconnected

**Comment by simonswiss** [comment]
Hey!

Have a look at the JIT troubleshooting section of the Tailwind CSS docs:

https://tailwindcss.com/docs/just-in-time-mode#troubleshooting

You might find the solution in there, possibly adding the `TAILWIND_MODE=watch` environment variable to the script starting your webpack devserver.

**Reply by huheng233** [reply]
TAILWIND_MODE=watch has been added before, but it is still the same

**Reply by huheng233** [reply]
I guess it is because the memory occupied by node is not recycled in time after the JIT mode is turned on
